packages/connector-google/README.md

```diff
@@ -1,2 +1,25 @@
-### Google Social Connector README
-placeholder
+# Google
+
+The Google connector provides a succinct way for your application to use Google’s OAuth 2.0 authentication system for users to login.
```

Comment:
I think `for users to login` is redundant and should be omitted

Reply:
updated.